intern.json

```diff
@@ -0,0 +1,59 @@
+{
+	"basePath": "./",
+	"environments": [
+		{ "browserName": "node" }
+	],
+	"maxConcurrency": 2,
+	"tunnel": "selenium",
+	"tunnelOptions": {
+		"hostname": "localhost",
+		"port": "4444"
+	},
+	"node": {
+		"reporters": [
+			{
+				"name": "runner"
+			},
+			{
+				"name": "jsoncoverage",
+				"options": {
+					"directory": "output/coverage"
+				}
+			},
+			{
+				"name": "htmlcoverage",
+				"options": {
+					"directory": "output/coverage/html"
+				}
+			}
+		],
+		"plugins": [
```

Comment:
Do we need all these plugins?

Reply:
Oh is this intern file used for the app and the scripts?

Reply:
Yes, I pulled it from the cli-test-intern's setup (minus the local config stuff).

https://github.com/dojo/cli-test-intern/blob/master/src/intern/intern.json

Reply:
Though now that I am building it down to js first, I can probably remove most of those. Let me check.

Reply:
I can only drop the `tsnode` plugin, so went ahead and did that.

Reply:
Can we run the tests in the browser (using browserstack or equivelent)? (i mean the dojo app ones not the script ones)